src/include/miopen/conv/problem_description.hpp

```diff
@@ -161,13 +161,13 @@ struct ProblemDescription
     std::size_t GetInStrideD() const { return GetD5(GetSpatialDims(), in.GetStrides()); }
     std::size_t GetInStrideH() const { return GetH5(GetSpatialDims(), in.GetStrides()); }
     std::size_t GetInStrideW() const { return GetW5(GetSpatialDims(), in.GetStrides()); }
-    std::string GetInLayout() const { return "NCHW"; }
+    std::string GetInLayout() const { return in.GetLayout(); }
```

Comment:
This function is used in `ProblemDescription::Serialize` that is used to generate database keys. Changing from "NCHW" to "nchw" would break find-db and, most likely, perf-db functionality.

Reply:
Sounds good thanks for reminding. Looks like to be consistent I will have to adopt all upper letters for layouts.